pkg/controller/migmigration/migrate.go

```diff
@@ -17,40 +17,27 @@ limitations under the License.
 package migmigration
 
 import (
-	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"time"
-)
 
-// Annotations
-const (
-	pvAnnotationKey             = "openshift.io/migrate-type"
-	migrateAnnotationValue      = "final"
-	migrateAnnotationKey        = "openshift.io/migrate-copy-phase"
-	stageAnnotationValue        = "stage"
-	stageAnnotationKey          = "openshift.io/migrate-copy-phase"
-	resticPvBackupAnnotationKey = "backup.velero.io/backup-volumes"
-)
-
-// Labels
-const (
-	pvBackupLabelKey   = "openshift.io/pv-backup"
-	pvBackupLabelValue = "true"
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // Backup resources.
 var stagingResources = []string{
+	"namespaces",
 	"persistentvolumes",
 	"persistentvolumeclaims",
 	"imagestreams",
 	"imagestreamtags",
 	"secrets",
 	"configmaps",
 	"pods",
+	"namespaces",
```

Comment:
Is namespaces supposed to be here twice?

Reply:
Actually, the namespace _work around_ is no longer needed so _namespaces_  no longer need to be included for stage backups.

Reply:
Fixed here: https://github.com/fusor/mig-controller/pull/196